src/neuromancer/dynamics/integrators.py

```diff
@@ -16,31 +16,27 @@ def __init__(self, block, interp_u=None, h=1.0):
         """
         Integration block.
 
-        :param block:
-        :param interp_u:
-        :param h:
+        :param block: (nn.Module or Callable) right hand side of the ODE system to be integrated
+        :param h: (float) integration step size
         """
         super().__init__()
         self.block = block  # block gives dx at this time step.
         self.in_features, self.out_features = block.in_features, block.out_features
-        self.interp_u = interp_u  # offline interpolant
         self.h = h
-        if self.interp_u is not None:
-            self.state = lambda x, tq, t, u: torch.cat([x, self.interp_u(tq, t, u)], dim=-1)
-        else:
-            self.state = lambda x, tq, t, u: x
+
+        if interp_u is not None:
+            warnings.warn('interp_u method is deprecated, it has no effect, and will be removed in the next release.')
```

Comment:
```suggestion
            warnings.warn('interp_u argument is deprecated, it has no effect, and will be removed in the next release.', FutureWarning)
```

https://docs.python.org/3/library/exceptions.html#FutureWarning